src/main/resources/assets/reignofnether/lang/zh_cn.json

```diff
@@ -1,32 +1,32 @@
-  "item.reignofnether.villager_unit_spawn_egg": "生成村民单位",
-  "item.reignofnether.zombie_villager_unit_spawn_egg": "生成僵尸村民单位",
-  "item.reignofnether.pillager_unit_spawn_egg": "生成掠夺者单位",
-  "item.reignofnether.vindicator_unit_spawn_egg": "生成卫道士单位",
-  "item.reignofnether.iron_golem_unit_spawn_egg": "生成铁傀儡单位",
-  "item.reignofnether.witch_unit_spawn_egg": "生成女巫单位",
-  "item.reignofnether.evoker_unit_spawn_egg": "生成唤魔者单位",
-  "item.reignofnether.enderman_unit_spawn_egg": "生成末影人单位",
-  "item.reignofnether.ravager_unit_spawn_egg": "生成劫掠兽单位",
-  "item.reignofnether.warden_unit_spawn_egg": "生成监守者单位",
-  "item.reignofnether.silverfish_unit_spawn_egg": "生成蠹虫单位",
-  "item.reignofnether.grunt_unit_spawn_egg": "生成咕哝者单位",
-  "item.reignofnether.brute_unit_spawn_egg": "生成猪灵蛮兵单位",
-  "item.reignofnether.headhunter_unit_spawn_egg": "生成猎头者单位",
-  "item.reignofnether.hoglin_unit_spawn_egg": "生成疣猪兽单位",
-  "item.reignofnether.blaze_unit_spawn_egg": "生成烈焰人单位",
-  "item.reignofnether.wither_skeleton_unit_spawn_egg": "生成凋灵骷髅单位",
-  "item.reignofnether.ghast_unit_spawn_egg": "生成恶魂单位",
-  "item.reignofnether.zombie_piglin_unit_spawn_egg": "生成僵尸猪灵单位",
-  "item.reignofnether.zoglin_unit_spawn_egg": "生成僵尸疣猪兽单位",
+  "item.reignofnether.zombie_unit_spawn_egg": "僵尸单位刷怪蛋",
+  "item.reignofnether.husk_unit_spawn_egg": "尸壳单位刷怪蛋",
+  "item.reignofnether.drowned_unit_spawn_egg": "溺尸单位刷怪蛋",
+  "item.reignofnether.skeleton_unit_spawn_egg": "骷髅单位刷怪蛋",
+  "item.reignofnether.stray_unit_spawn_egg": "流浪者单位刷怪蛋",
+  "item.reignofnether.creeper_unit_spawn_egg": "苦力怕单位刷怪蛋",
+  "item.reignofnether.spider_unit_spawn_egg": "蜘蛛单位刷怪蛋",
+  "item.reignofnether.poison_spider_unit_spawn_egg": "洞穴蜘蛛单位刷怪蛋",
+  "item.reignofnether.villager_unit_spawn_egg": "村民单位刷怪蛋",
+  "item.reignofnether.zombie_villager_unit_spawn_egg": "僵尸村民单位刷怪蛋",
+  "item.reignofnether.pillager_unit_spawn_egg": "掠夺者单位刷怪蛋",
+  "item.reignofnether.vindicator_unit_spawn_egg": "卫道士单位刷怪蛋",
+  "item.reignofnether.iron_golem_unit_spawn_egg": "铁傀儡单位刷怪蛋",
+  "item.reignofnether.witch_unit_spawn_egg": "女巫单位刷怪蛋",
+  "item.reignofnether.evoker_unit_spawn_egg": "唤魔者单位刷怪蛋",
+  "item.reignofnether.enderman_unit_spawn_egg": "末影人单位刷怪蛋",
+  "item.reignofnether.ravager_unit_spawn_egg": "劫掠兽单位刷怪蛋",
+  "item.reignofnether.warden_unit_spawn_egg": "监守者单位刷怪蛋",
+  "item.reignofnether.silverfish_unit_spawn_egg": "蠹虫单位刷怪蛋",
+  "item.reignofnether.grunt_unit_spawn_egg": "咕哝者单位刷怪蛋",
+  "item.reignofnether.brute_unit_spawn_egg": "猪灵蛮兵单位刷怪蛋",
+  "item.reignofnether.headhunter_unit_spawn_egg": "猎头者单位刷怪蛋",
+  "item.reignofnether.hoglin_unit_spawn_egg": "疣猪兽单位刷怪蛋",
+  "item.reignofnether.blaze_unit_spawn_egg": "烈焰人单位刷怪蛋",
+  "item.reignofnether.wither_skeleton_unit_spawn_egg": "凋灵骷髅单位刷怪蛋",
+  "item.reignofnether.ghast_unit_spawn_egg": "恶魂单位刷怪蛋",
+  "item.reignofnether.zombie_piglin_unit_spawn_egg": "僵尸猪灵单位刷怪蛋",
+  "item.reignofnether.zoglin_unit_spawn_egg": "僵尸疣猪兽单位刷怪蛋",
 
   "sounds.reignofnether.under_attack": "遭到袭击",
```

Comment:
The exclamation mark isn't consistent in zh_cn and zh_tw

Reply:
> The exclamation marks aren't consistent in `zh_cn` and `zh_tw`. And according to `en_us`, there should be **exclamation marks** after `"遭到攻击"` and `"游戏获胜"`. Also, in my opinion, literal translations of `spawn_egg` are not elegant or consistent with `en_us`.

The suggestion to use exclamation marks is appreciated but falls outside the scope of this PR, which is centered on translation standardization. 
According to the translation standardization guidelines listed on the Minecraft Wiki and the historical translation changes for "spawn eggs", the translation shifted from "生成<生物>" to "<生物>刷怪蛋" as of version 1.18 18w20a. Consequently, the term "生成" should not be used when referring to "spawn egg," as it would result in an incorrect translation.
This approach is not merely a "literal translation"; it respects community-recognized standards that should be adhered to, even if it is not officially part of Minecraft content.
Refer to: [Minecraft Wiki:译名标准化](https://zh.minecraft.wiki/w/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96), [刷怪蛋](https://zh.minecraft.wiki/w/%E5%88%B7%E6%80%AA%E8%9B%8B)